api/topology/src/test/java/quarks/test/topology/PlumbingTest.java

```diff
@@ -601,4 +602,66 @@ public void testParallel() throws Exception {
 //            actDuration < 0.5 * expMinSerialDuration);
 //    }
 
-}
+    @Test
+    public void testGate() throws Exception {
+        // Timing variances on shared machines can cause this test to fail
```

Comment:
Hi @dlaboss 
Sorry for that. At first time I was planning to make test with PlumbingStreams.blockingDelay (similar to the testBlockingDelay) However, plans changed but the line did not erased.
I will erase immediately "assumeTrue" line in two Testcase